src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
> What I meant was the sRGB primaries with any other white point. No adaption, just a different combination.

I am not sure I understand what you are trying to say. The Srgb primaries are defined with the D65 white point. I have not seen any other definitions for the primaries. For use with any other white point, chromatic adaptation for the primaries is required.